src/encoded/static/components/objectutils.js

```diff
@@ -618,18 +618,9 @@ export class ImageWithFallback extends React.Component {
         });
     }
 
-    // Display default "not found" image for non-existent image src
-    onError() {
```

Comment:
We may have to discuss this in person. IE11 throws a permission error always. I skipped this check and let the error fall through.